src/test/kotlin/ar/edu/unahur/obj2/socios/ClienteTest.kt

```diff
@@ -1,8 +1,87 @@
 package ar.edu.unahur.obj2.socios
 
 import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.matchers.shouldBe
+
 
 class ClienteTest : DescribeSpec({
   describe("Un/a cliente") {
+    var cliente = Cliente(Resfriado, LasLauchas,1000)
+    describe("Estado de animo del cliente"){
+      cliente.estadoDeAnimo.shouldBe(Resfriado)
+    }
+    describe("Cantidad de plata en el bolsillo del cliente"){
+      cliente.dineroEnBolsillo.shouldBe(1000)
+    }
+    describe("Donde vive el cliente"){
+      cliente.viveEn.shouldBe(LasLauchas)
+    }
+    describe("Propina de un pedido de 100 pesos dependiendo su estado de animo"){
+      it("Cuando esta enojado ") {
+        cliente.viveEn = LasTorres
+        cliente.estadoDeAnimo = Enojado
+        cliente.propina(100).shouldBe(0)
+      }
+      it("Cuando esta feliz") {
+        cliente.viveEn = LasTorres
+        cliente.estadoDeAnimo = Feliz
+        cliente.propina(100).shouldBe(25)
+      }
+      it("Cuando esta indiferente") {
+        cliente.viveEn = LasTorres
+        cliente.estadoDeAnimo = Indiferente
+        cliente.propina(100).shouldBe(1000)
+      }
+      it("Cuando esta resfriado") {
+        cliente.viveEn = LasTorres
+        cliente.estadoDeAnimo = Resfriado
+        cliente.propina(100).shouldBe(100)
+      }
+    }
```

Comment:
Una única observación, la asignación de `LasTorres` la podrian poner arriba así no la repiten en cada `it`:

```suggestion
    describe("Propina de un pedido de 100 pesos dependiendo su estado de animo"){
      cliente.viveEn = LasTorres
      
      it("Cuando esta enojado ") {
        cliente.estadoDeAnimo = Enojado
        cliente.propina(100).shouldBe(0)
      }
      it("Cuando esta feliz") {
        cliente.estadoDeAnimo = Feliz
        cliente.propina(100).shouldBe(25)
      }
      it("Cuando esta indiferente") {
        cliente.estadoDeAnimo = Indiferente
        cliente.propina(100).shouldBe(1000)
      }
      it("Cuando esta resfriado") {
        cliente.estadoDeAnimo = Resfriado
        cliente.propina(100).shouldBe(100)
      }
    }
```

Reply:
No nos avivamos de eso, gracias por la sugerencia !!